.github/workflows/upload-go-master.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           # See https://golang.org/cmd/link/ for info on -w (omit the DWARF symbol table) and -s (omit the symbol table and debug information)
           # We use those flags to get a smaller compiled binary for faster downloads.
           go build -ldflags "-w -s -X github.com/SAP/jenkins-library/cmd.GitCommit=${GITHUB_SHA} \
+                  -X github.com/SAP/jenkins-library/cmd.GitTag=${GITHUB_REF_NAME} \
```

Comment:
not sure if I understand this correctly ...

The Piper Version tag is created weekly and not known at build time ...